test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -150,7 +150,8 @@ public static void ClearProfilerEnvironmentVariables()
             int aspNetCorePort,
             int? statsdPort,
             string processPath,
-            StringDictionary environmentVariables)
+            StringDictionary environmentVariables,
+            bool ignoringProfilerProcesses = false)
```

Comment:
nit: should this be a command, ie. `ignoreProfilerProcesses`?

Reply:
I didn't do a method for this just because the default behavior is to set `DD_PROFILER_PROCESSES` env var and I want to skip that in this case. If I want to do that in a method I need to store that state and check it in this method as well. It was simpler adding this default argument.

Reply:
Sorry, I wasn't clear here, I actually just meant renaming the var 🙂  e.g.
```suggestion
            bool ignoreProfilerProcesses = false)
```

Reply:
oh sorry I misunderstood that completely 😅. accepting your suggestion. Thank you!

Reply:
Don't forget to update the usages in the method too 😉